styles.css

```diff
@@ -1,504 +1,1051 @@
+    padding-left: 1px;
+    background: #fff;
+    border-top-right-radius: 100px;
+  }
+
+  .section2-ul {
+    display: flex;
+    flex-direction: column;
+    align-items: flex-start;
+    width: 98%;
+    background-color: #fff;
+    margin-top: 222px;
+    align-self: left;
+  }
+
+  .section2-li {
+    width: 92%;
+    display: flex;
+    flex-direction: row;
+    justify-content: space-between;
+    margin-top: 5px;
+    margin-bottom: 23px;
+  }
+
+  .section2-h2 {
+    font-family: "Poppins", sans-serif;
+    font-style: normal;
+    font-weight: 600;
+    font-size: 20px;
+    align-items: center;
+    color: #000;
+  }
+
+  .icon2 {
+    position: relative;
+    top: 23px;
+    right: 28px;
+  }
+
+  .languages {
+    display: flex;
+    flex-direction: row;
+    justify-content: space-between;
+    width: 80%;
+  }
+
+  .lan {
+    text-align: center;
+    width: 129px;
+    height: 108px;
+    padding-top: 18px;
+    padding-right: -8px;
+    background: #f7f7f9;
+    border-radius: 10px;
+  }
+
+  .hr {
+    border-bottom: 2px solid rgb(188, 188, 188);
+    position: relative;
+    top: 44px;
+    width: 92%;
+  }
+
+  /* footer */
+
+  footer {
+    display: flex;
+    flex-direction: column;
+    align-items: center;
+    gap: 20px;
+    width: 100%;
+    height: 750px;
+    background-color: #6070ff;
+    border-top-left-radius: 100px;
+    background-image: url(./images/contact\ form\ background\ shapes@2x.png);
+    background-size: 100%;
+    background-position-y: 80px;
+    background-repeat: no-repeat;
+  }
+
+  .contact_paragraph {
+    font-family: "Poppins", sans-serif;
+    font-style: normal;
+    font-weight: 400;
+    font-size: 20px;
+    line-height: 28px;
+    text-align: center;
+    width: 540px;
+    color: #ebebff;
+    margin-top: -31px;
+  }
+
+  .contact_title {
+    font-family: "Poppins", sans-serif;
+    font-style: normal;
+    font-weight: 700;
+    font-size: 35px;
+    line-height: 52px;
+    color: #fff;
+    margin-top: 100px;
+  }
+
+  .form-section {
+    display: flex;
+    flex-direction: column;
+  }
+
+  .form-section input {
+    width: 400px;
+    margin: 10px 30px;
+    height: 32px;
+    padding: 5px 14px;
+    font-size: 17px;
+    line-height: 20px;
+    border: 1px solid #cfd8dc;
+    border-radius: 8px;
+  }
+
+  #message {
+    width: 400px;
+    height: 123px;
+    font-style: normal;
+    font-family: "Poppins", sans-serif;
+    font-weight: 400;
+    font-size: 15px;
+    line-height: 32px;
+    padding: 5px 14px;
+    border: 1px solid #cfd8dc;
+    border-radius: 8px;
+    margin: 10px 30px 20px 30px;
+  }
+
+  .contact_button {
+    width: 133px;
+    height: 45px;
+    border: 1px solid #6070ff;
+    border-radius: 8px;
+    font-family: "Poppins", sans-serif;
+    font-style: normal;
+    font-weight: 500;
+    font-size: 17px;
+    line-height: 24px;
+    letter-spacing: 0.03em;
+    color: #396df2;
+    position: relative;
+    left: 27px;
+    margin-bottom: 40px;
+  }
```

Comment:
- [ ] Your work on the contact me section is fantastic :+1:. However, I kindly request that you align the button in the middle of the form. This is essential to ensure that the design remains balanced and visually appealing to users.

**Your design**

![Screenshot from 2023-03-31 18-48-20](https://user-images.githubusercontent.com/105117832/229251532-27fc2df5-8098-40a8-b087-143d5c707c3a.png)

**Figma**

![Screenshot from 2023-03-31 18-48-41](https://user-images.githubusercontent.com/105117832/229251599-661788de-eab0-4428-8b65-6f2e14db5d0f.png)